glossary.tex

```diff
@@ -268,6 +268,32 @@
                worden sind, sowie die die Menge der unbearbeiteten
                Änderungswünschen (Featurerequests).}
 }
+\newglossaryentry{Peer-to-Peer}
+{
+  name=Peer-to-Peer,
+  description={Begriff aus der Computer Netzwerkstruktur. Es beschreibt die
+               Verbindungsart zwischen zwei Computern in einem Netzwerk oder
+               Internet. Dabei kommunizieren zwei Teilnehmer über eine direkte
+               Verbindung miteinander, ohne das einer der beiden eine
+               Sonderstellung (Server) einnimmt.}
+}
+\newglossaryentry{Partitionen}
+{
+  name=Partitionen,
+  description={Eine Partitione definiert einen logischen Bereich der im
```

Comment:
das passt nicht ganz zum kontext der datenbanken.

Partitionierung im allgemeinen beschreibt das einteilen vorhandener ressourcen in blöcke.

Am bekanntesten ist dies bei Festplatten. hier teilt man vorhandenen speicher in mehrere logische bereiche ein. Bei Datenbanken gibt es ebenfalls partitioning. hier hat man zumeist eine viel zu fette tabelle und irgendeinen tollen identifer, z.b. ein datum. wenn man hier von partitioning spricht dann erstellt man mehrere versteckte tabellen, z.b. eine pro tag oder pro stunde. nach aussen hin ist erstmal nur die eine große sichtbar, die setzt sich aber aus den anderen zusammen. wenn man dann selects draufprügelt mit WHERE clause die nach datum sucht dann weis das DBMS in welcher untertabelle/partition es schauen muss, geht somit wesentlich schneller weil nicht der index einer mega tabelle gelesen werden muss. oder noch schlimmer ein sequential scan